package.json

```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "@commitlint/cli": "^17.7.1",
     "@commitlint/config-conventional": "^17.7.0",
-    "@ridedott/eslint-config": "^2.23.1",
+    "@ridedott/eslint-config": "^2.23.75",
```

Comment:
Are this changes related to the Node bump?

Reply:
Yeah it seems so. This rule started breaking in this PR.

Reply:
Reverted in https://github.com/ridedott/merge-me-action/pull/1819/commits/ca88ea0e068f7cc0e20557a519db9bb8d81dcec9.